### DISCUSSION TITLE
TuringCourses/others/numerical_linear_algebra/

### DISCUSSION THREAD

**Discussion by giscus** [ZJU-Turing/TuringCourses#59] [General]
# TuringCourses/others/numerical_linear_algebra/

图灵班学习指南

https://zju-turing.github.io/TuringCourses/others/numerical_linear_algebra/

**Comment by ZhouTimeMachine** [comment]
数值类课程主要有三门，分析、代数、数值解，全部学完就可以成仙了（笑）
